tests/m2m_signals/tests.py

```diff
@@ -54,25 +55,23 @@ def tearDown(self):
             self.m2m_changed_signal_receiver, Person.friends.through
         )
 
-    def test_m2m_relations_add_remove_clear(self):
-        expected_messages = []
-
-        # Install a listener on one of the two m2m relations.
+    def initialize_signal_car(self, add_default_parts_before_set_signal=False):
```

Comment:
Make it a "private" method: `_initialize_signal_car` just to make it obvious it's not a test method.